fluent/toInterfaceValue_test.go

```diff
@@ -0,0 +1,51 @@
+package fluent_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-ipld-prime/fluent"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
+	"github.com/ipld/go-ipld-prime/node/basicnode"
+)
+
+var roundTripTestCases = []struct {
+	desc  string
```

Comment:
nit: usually you'd name this `name`, since it's the sub-test name, not really its description